stdlib/public/core/MutableCollection.swift

```diff
@@ -238,6 +238,8 @@ extension MutableCollection {
   ///   the range must be valid indices of the collection.
   ///
   /// - Complexity: O(1)
+  // FIXME: somehow mark unavailable/deprecated/obsoleted for SwiftStdlib < 5.5
+  // @available(swift, unavailable)
```

Comment:
I would expect that making this unconditionally unavailable (but still present) while adding the conditional extension below (with `_aeic`) would make things work.

Reply:
That does work, but I wasn't sure whether it would be valid. Since we won't try to make it into the 5.5 branch, I'm less concerned.